client/signup/steps/difm-page-picker/style.scss

```diff
@@ -0,0 +1,13 @@
+.signup__step.is-difm-page-picker {
+	.step-wrapper.is-horizontal-layout {
+		margin-top: 25px;
+		@media ( min-width: 600px ) {
```

Comment:
We can use the existing mixin `@include break-small` here.

Reply:
I think you missed adding the mixin here. 🙂 
Also, the build fails:
<img width="608" alt="image" src="https://user-images.githubusercontent.com/5436027/158809992-234246d0-1d3c-4f1a-a433-264ce8cb9fe7.png">